backend/spec/models/option_spec.rb

```diff
@@ -22,11 +22,17 @@
     end
 
     describe 'when many options are set to correct' do
-      it { expect(option.errors.full_messages).to match(['Single choice questions should have no more than one correct option.']) }
+      it { expect(option.errors.full_messages).to match(['Name has already been taken', 'Single choice questions should have no more than one correct option.']) }
```

Comment:
this spec is not related to the issue that you are working on, maybe you should change the option name, so this spec wouldn't change